static-site/src/pages/groups.jsx

```diff
@@ -4,23 +4,50 @@ import { SmallSpacer, HugeSpacer, FlexCenter } from "../layouts/generalComponent
 import * as splashStyles from "../components/splash/styles";
 import { fetchAndParseJSON } from "../util/datasetsHelpers";
 import DatasetSelect from "../components/Datasets/dataset-select";
+import ListResources from "../components/ListResources/index";
 import { GroupCards } from "../components/splash/groupCards";
 import GenericPage from "../layouts/generic-page";
 import { UserContext } from "../layouts/userDataWrapper";
 import { DataFetchErrorParagraph } from "../components/splash/errorMessages";
 import { groupsTitle, GroupsAbstract } from "../../data/SiteConfig";
 
-const datasetColumns = ({isNarrative}) => [
+const resourceListingCallback = async (setDataFetchError) => {
+  const sourceUrl = "/charon/getAvailable?prefix=/groups/";
+
+  try {
+    const response = await fetch(sourceUrl, {headers: {accept: "application/json"}});
```

Comment:
FYI - the charon API doesn't use content negotiation. No need to change, but just so you know this header isn't doing anything ;)

Reply:
fixed since i was in there anyway.